core/src/themes/test/colors/index.html

```diff
@@ -320,26 +320,26 @@ <h3 class="header">Background 16%</h3>
     </ion-app>
 
     <script>
-      const themePicker = document.querySelector('.theme-picker');
+      const palettePicker = document.querySelector('.palette-picker');
 
-      themePicker.addEventListener('ionChange', (ev) => {
-        const theme = event.detail && event.detail.value;
-        toggleTheme(theme);
+      palettePicker.addEventListener('ionChange', (ev) => {
+        const palette = event.detail && event.detail.value;
+        togglePalette(palette);
       });
 
-      function toggleTheme(theme) {
-        let cssFile = `/css/themes/${theme}.always.css`;
-        if (theme === 'default') {
+      function togglePalette(palette) {
+        let cssFile = `/css/palettes/${palette}.always.css`;
+        if (palette === 'default') {
           // TODO FW-5862 update this to not
           // load a file when default is set. The light
-          // theme is automatically set when importing ionic.bundle.css
-          cssFile = `/src/themes/test/${theme}.css`;
+          // palette is automatically set when importing ionic.bundle.css
+          cssFile = `/src/palettes/test/${palette}.css`;
```

Comment:
It seems like this file is still at `/src/themes/test/default.css`

Reply:
Good catch, I addressed that here: https://github.com/ionic-team/ionic-framework/pull/29149/commits/ff52cc22f9fc3efcfbf46fdc3b22891a9f8428d3 & simplified just saying what we are explicitly importing, since we don't really need to use the `palette` to generate the path.